Library/Homebrew/formula_cellar_checks.rb

```diff
@@ -217,10 +217,7 @@ def audit_installed
     audit_check_output(check_generic_executables(formula.bin))
     audit_check_output(check_non_executables(formula.sbin))
     audit_check_output(check_generic_executables(formula.sbin))
-    audit_check_output(check_shadowed_headers)
     audit_check_output(check_easy_install_pth(formula.lib))
-    audit_check_output(check_openssl_links)
-    audit_check_output(check_python_framework_links(formula.lib))
```

Comment:
Shouldn't these checks, that have been relocated to an OS-specific implementation file, be removed from this file?